lib/schemas.py

```diff
@@ -23,59 +35,77 @@ class VideoResponse(MediaResponse):
-    body: GenericItem
+    request_id: Union[str, int, float]
+    items: List[GenericItem]  # to support batch, this is a list (can be only 1 item)
     model_name: str
+    model_parameters: Optional[Dict] = (
+        {}
+    )  # optional parameters specific to this model call (i.e. threshold)
     retry_count: int = 0
+    callback_url: Optional[str] = None
 
 
 def parse_input_message(message_data: Dict) -> Message:
-    if 'body' not in message_data or 'model_name' not in message_data:
-        raise PrestoBaseException("Invalid message data: message should at minimum include body and model_name"
-                                  , 422)
+    if "body" not in message_data or "model_name" not in message_data:
+        raise PrestoBaseException(
+            "Invalid message data: message should at minimum include body and model_name",
+            422,
+        )
 
-    body_data = message_data['body']
-    model_name = message_data['model_name']
-    result_data = body_data.get('result', {})
+    body_data = message_data["body"]  # TODO: this becomes the 'items' list
+    model_name = message_data["model_name"]
+    result_data = body_data.get("result", {})
+    # TODO: validate callback_url
 
     try:
-        presto_model_class_name = model_name.replace('__', '.')  # todo don't love this line of code
-        model_class = get_class('lib.model.', presto_model_class_name)
+        presto_model_class_name = model_name.replace(
+            "__", "."
+        )  # todo don't love this line of code
+        model_class = get_class("lib.model.", presto_model_class_name)
     except Exception as e:
-        raise PrestoBaseException(f"Error loading model {model_name}, model_name is not supported: {e}", 404) from e
+        raise PrestoBaseException(
+            f"Error loading model {model_name}, model_name is not supported: {e}", 404
+        ) from e
 
-    model_class.validate_input(body_data)  # will raise exceptions in case of validation errors
+    model_class.validate_input(
+        body_data
+    )  # will raise exceptions in case of validation errors
     # parse_input_message will enable us to have more complicated result types without having to change the schema file
-    result_instance = model_class.parse_input_message(body_data)  # assumes input is valid
+    result_instance = model_class.parse_input_message(
+        body_data
+    )  # assumes input is valid
 
     # TODO: the following is a temporary solution to handle the case where the model does not have a
     # parse_input_message method implemented but we must ultimately implement parse_input_message and
     # validate_input in all models. ticket: https://meedan.atlassian.net/browse/CV2-5093
-    if result_instance is None:  # in case the model does not have a parse_input_message method implemented
-        if 'yake_keywords' in model_name:
+    if (
+        result_instance is None
+    ):  # in case the model does not have a parse_input_message method implemented
```

Comment:
imo this is not a more readable formatting than what was before lol, any chance to manually make this pretty or revert this line back? 

@skyemeedan is there a way to opt out of formatting everything in the future? aside from cases like these, it makes it difficult to find out what the actual diffs are.

Reply:
I agree it is hard to read when substantive and formatting changes are mixed together.  Which is why I'm a huge fan of configuring my editor to use a strict PEP-8 syntax formatting hook (especially if we all do it!)   I find it saves a huge amount of time  not having to manually mess with formatting or linting fixes as much.  

I could make a separate PR to first apply the formatting to the whole repo? ;-)  
I can also just move the comment up a line and it should fit within the line length